### ISSUE TITLE
Move the Registry database to the US-East-1a availability zone.

### ISSUE THREAD

**Issue by tzouris** [mlaa/commons#630]
The RDS instance currently exists in the US-East-1b AZ.

Steps:
-Sever the connection with Registry.
-Take a snapshot of the db.
-Restore the snapshot as a new RDS instance.
-Update the registry config file.
-Restore connection to Registry

**Comment by tzouris** [comment]
The journey:
-Commented out the host line in the registry database.php file.
-Took a snapshot of the db.
-Renamed the old RDS instance.
-Restored the snapshot as a new RDS instance with the same name.
-Didn't have to update the registry database.php file with the new endpoint - the endpoint was the same
-Restored the connection to the db
-Stopped the old db temporarily. Will delete it within 7 days.

Tested the registration button and CoManage looks like it's working.

Amazon tried to fool me into purchasing an RDS instance with double the cost by defaulting to the m5.xlarge instance instead of the m5.large (the one we had before). I didn't notice that at first, so I had to repeat the restore process again. Other options were not correct either, but I caught those during the first restore procedure (VPC security groups, subnet groups etc.)
